src/main/java/games/strategy/engine/framework/ui/GameChooserModel.java

```diff
@@ -90,10 +90,7 @@ public GameChooserEntry get(final int i) {
       } catch (final InterruptedException e) {
         Thread.currentThread().interrupt();
       } catch (final ExecutionException e) {
-        // ExecutionException contains no useful information; it's simply an adapter to tunnel
-        // exceptions thrown by tasks through the Executor API. Log the cause only to reducase
-        // stack trace frames.
-        ClientLogger.logError("Failed to parse a map", e.getCause());
+        ClientLogger.logError("Failed to parse a map", e);
```

Comment:
Should we also do a `: " + e.getMessage(), e);` here as well?

Reply:
I don't see why not. :smile:  c7a9dd1ec

What do you think of _always_ including `e.getMessage()` in the log message now that we're displaying it to the user?  Then we could bury that formatting in the `ClientLogger` methods that accept a `Throwable`, and remove the responsibility for doing that from the call site.